### ISSUE TITLE
Dilated ResNet in SFNet?

### ISSUE THREAD

**Issue by wondervictor** [lxtGH/SFSegNets#46]
Hi, I find that the SFNet w/ ResNet-18 has adopt dilated convolutions in C4 and C5:
https://github.com/lxtGH/SFSegNets/blob/773c7b75a3ee426f21d0e601568462f952d373f4/network/sfnet_resnet.py#L202
However, it hasn't been mentioned in the paper or the other implementation [torchcv](https://github.com/donnyyou/torchcv).
I'm confused about it.

**Comment by lxtGH** [comment]
@wondervictor Hi! Note that this is reproduce of our SFnet which achieve better reasuts than paper. You can turn the dilation off. I remember it only drop a little after longer training.

**Comment by wondervictor** [comment]
Besides, another setting is strange to me.
see:
https://github.com/lxtGH/SFSegNets/blob/ce97d2e7dfd7cf3f3d2af7b0c31e23e51df6b182/network/sfnet_resnet.py#L53
and 
https://github.com/lxtGH/SFSegNets/blob/ce97d2e7dfd7cf3f3d2af7b0c31e23e51df6b182/network/sfnet_resnet.py#L79
The normalization layers in `fpn_out` and `conv_last` are `nn.BatchNorm2d` but not `SyncBatchNorm` due to unspecific parameter `normal_layer` for function `conv3x3_bn_relu`.
I'm concerned about whether it is a special setting or a bug.

**Comment by lxtGH** [comment]
It may be a bug but it seems that it dose not effect the performance.

**Comment by lxtGH** [comment]
Time to close this issue. If you have more question. please re-open it.
